src/illuminatio/host.py

```diff
@@ -1,50 +1,63 @@
+"""
+file for all kinds of hosts
```

Comment:
Could you add a better comment and stat what a `host` is in the context of illuminatio?